canard.c

```diff
@@ -262,13 +262,13 @@ void canardHandleRxFrame(CanardInstance* ins, const CanardCANFrame* frame, uint6
         (frame->id & CANARD_CAN_FRAME_ERR) != 0 ||
         (frame->data_len < 1))
     {
-        return;     // Unsupported frame, not UAVCAN - ignore
+        return -CANARD_ERROR_RX_INCOMPATIBLE_PACKET;     // Unsupported frame, not UAVCAN - ignore
```

Comment:
Here and elsewhere in this function, these little comments that follow the return code are not useful anymore, since they basically repeat the information that is already presented by the name of the error code. I suggest dropping them.